SU-HSDO/suhumsci
# updated layout builder patch

# READY FOR REVIEW

# Summary
- Removed layout builder patch. the patch has been applied to 8.6.x branch
- https://www.drupal.org/project/drupal/issues/2936358#comment-12705127

# Needed By (Date)
- Whenever no rush

# Urgency
- low

# Steps to Test

1. Checkout this branch
2. `rm -rf docroot/core`
3. `composer update --prefer-source`
4. `blt drupal:sync --site=archaeology --sync-files --partial`
5. Verify node pages look the same as prod and views look the same

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
